src/index.ts

```diff
@@ -23,6 +23,7 @@ import { YosemiteDecimal, French, UIAA, Font, VScale, Ewbank } from './scales'
 // Nordic - NOR
 // Brazilian
 // Kurtyka (Poland)
+// Norwegian (aka Scandinavian)
```

Comment:
Could you remove this comment and the one for NOR? This is a list of grades we don't have yet.